userservice/src/main/java/com/epam/app/repository/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.app.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.epam.app.model.User;
+
+public interface UserRepository extends JpaRepository<User, Integer>{
```

Comment:
are you able to autowire this is service without annotating the class with @Repository?

Reply:
Yes it is working.

Reply:
okay